bower.json

```diff
@@ -24,7 +24,6 @@
     "polymer": "polymer/polymer#^1.1.0",
     "iron-a11y-keys-behavior": "polymerelements/iron-a11y-keys-behavior#^1.0.0",
     "iron-icon": "polymerelements/iron-icon#^1.0.0",
-    "iron-icons": "polymerelements/iron-icons#^1.0.0",
```

Comment:
There is no `iron-iconset-svg#^1.1.3`. Do we need something particular in that release, or does the classic `#^1.0.0` work?